patches/amd-mainline/HIPIFY/0001-Make-hipify-clang-buildable-with-an-LLVM-configured-.patch

```diff
@@ -85,5 +85,5 @@ index 1b51a16e..d2055a01 100755
    if(LLVM_PACKAGE_VERSION VERSION_EQUAL "16.0.0" OR LLVM_PACKAGE_VERSION VERSION_GREATER "16.0.0")
      if(MSVC)
 -- 
-2.43.0
+2.47.1.windows.2
```

Comment:
Not sure if the diffs in these patch files are load bearing or if should revert them. I just committed what my repository looked like after running `bash .\build_tools\save_patches.sh m/amd-mainline HIPIFY amd-mainline`

Reply:
I don't have a strong opinion here.